rcsb/utils/dictionary/DictionaryApiProviderWrapper.py

```diff
@@ -81,9 +81,9 @@ def getApiByName(self, databaseName, **kwargs):
         else:
             dictLocators = [self.__cfgOb.getPath(configLocator, sectionName=self.__configName) for configLocator in self.__dictLocatorMap[databaseName]]
             # Example dictLocators for databaseName 'pdbx_comp_model_core':
-            #  ['https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/mmcif_pdbx_v5_next.dic',
-            #   'https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/dist/rcsb_mmcif_comp_model_ext.dic',
-            #   'https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionary_files/reference/mmcif_ma.dic']
+            #  ['https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_pdbx_v5_next.dic',
+            #   'https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_comp_model_ext.dic',
+            #   'https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_ma.dic']
```

Comment:
@piehld This would be `mmcif_ma_ext.dic`.

Reply:
@brindakv Doh. Yes I'll create a new PR. Thanks for catching this. Maybe I'll leave the version alone though, since it's just comments that are updated?

Reply:
Sounds good @piehld